app/database/models/server/index.ts

```diff
@@ -26,5 +26,6 @@ export {default as TeamModel} from './team';
 export {default as TeamSearchHistoryModel} from './team_search_history';
 export {default as TermsOfServiceModel} from './terms_of_service';
 export {default as ThreadModel} from './thread';
+export {default as ThreadInTeamModel} from './team_thread';
```

Comment:
Can we rename the file to `threads_in_team` to match the model name, or change the model name, you decide